docs/user-guide/using-jwt-token-with-apiml.md

```diff
@@ -0,0 +1,223 @@
+-->
+
+### Token-based Login Flow and Request/Response Format
+
+The following sequence describes how authentication through JWT tokens works:
+
+1. The API client obtains a JWT token by using the POST method on the `/auth/login` endpoint of the API service that requires a valid user ID and password.
+
+2. The API service then calls an *authentication provider* that returns a JWT token that contains the user ID claim in the HTTP cookie named `apimlAuthenticationToken` with attributes `HttpOnly` and `Secure`.
+
+3. The API client stores the JWT token or cookie and sends the token with every request as a cookie with the name `apimlAuthenticationToken`.
+
+## Obtaining a token
+
+To obtain a token, call the endpoint with the credentials for either basic authentication or the client certificate.
+
+<!-- Check the previous statement for accuracy. -->
+
+- The full path for API ML is:```/gateway/auth/login```
+
+- The full URL is the base URL of the API service plus `/auth/login`. If the application has the base URL with `/api/v1`, the full URL could have the format: `https://hostname:port/api/v1/auth/login`.
+
+- The credentials are provide in the JSON request:
+
+    ```json
+    {
+        "username": "...",
+        "password": "..."
+    }
+    ```
+
+- Successful login returns RC `204`, and an empty body with the token in the `apimlAuthenticationToken` cookie.
+
+- Failed authentication returns RC `401` without `WWW-Authenticate`.
+
+**Example:**
+
+```bash
+curl -v -c - -X POST "https://localhost:10080/api/v1/auth/login" -d "{ \"username\": \"zowe\", \"password\": \"zowe\"}"
+```
+
+```http
+POST /api/v1/auth/login HTTP/1.1
+Accept: application/json, */*
+Content-Length: 40
+Content-Type: application/json
+
+{
+    "username": "zowe",
+    "password": "zowe"
+}
+
+HTTP/1.1 204
+Set-Cookie: apimlAuthenticationToken=eyJhbGciOiJSUzI1NiJ9...; Path=/; Secure; HttpOnly
+```
+
+### Making an authenticated request
+
+You can send a JWT token with a request in two ways:
+
+* Allow the API client to pass the JWT token as a cookie header
+* Pass the JWT token in the `Authorization: Bearer` header
+
+:::tip
+The first option (using a cookie header) is recommended for web browsers with the attributes `Secure` and `HttpOnly`.
+Browsers store and send cookies automatically.
+Cookies are present on all requests, including those coming from DOM elements, and are compatible with web mechanisms such as CORS, SSE, or WebSockets.
+
+Cookies are more diffcult to support in non-web applications.
+Headers, such as `Authorization: Bearer`, can be used in non-web applications. Such headers, however, are difficult to use and secure in a web browser.
+The web application needs to store these headers and attach these headers to all requests where headers are required.
+
+Note that the API service needs to support both of them <!-- What is "both of them" --> so the API clients can use what makes more sense for them.
```

Comment:
Remov this line. It's not relevant for this article. It relates to the extensions.